### DISCUSSION TITLE
Tasks Links (Relationships)

### DISCUSSION THREAD

**Discussion by newkis** [Leantime/leantime#1374] [Ideas]
It would be nice, if user had possibility to link tasks with some types of link, if tasks are links by something. For example, user lays out some project on different tasks. And some tasks have to be done earlier than others, because the result of this tasks is necessary for next task's steps. And if it was possibility to link tasks, such as one task depends from other, it would be nice, because user can see on Gantt chart addiction of list tasks, which would be shown on time stamp.
May be it would be nice to enter different type of links between tasks, such as: link with, duplicated by, duplicate, blocking by, blocked, next, previous, copy from, copy to. In Redmine it realized.

**Comment by xhk416x** [comment]
came here to request this. +1

**Reply by marcelfolaron** [reply]
Tasks can already be linked via the subtask feature. Are you looking for something different?

**Reply by xhk416x** [reply]
The different types of links, like:
- Next
- Previous
- Blocks
- Blocked By

And I acknowledge I could just not know how to do it.

**Comment by marcelfolaron** [comment]
Ah I See. Thanks for clarifying.
